### ISSUE TITLE
LaTeX font options

### ISSUE THREAD

**Issue by sswatson** [sciapp/gr#93]
I'm wondering whether it would be helpful to allow the user to change the LaTeX font by injecting code into the header of the LaTeX file being compiled. That way, users could get their math font to match their text font.

I hardly have the full view, but it seems that maybe this could be implemented by changing a [few lines](https://github.com/sciapp/gr/blob/89cac500338180558e5124b3b7a214a2112d3dc5/lib/gr/gr.c#L8761) in `latex2image` in `gr.c`.

**Comment by jheinen** [comment]
We could think of site-specific LaTeX prologues, given in an environment or a configuration file. Would this be acceptable for you?

**Comment by sswatson** [comment]
Yes, that sounds like a good solution. Ultimately I'd like to be able to do something like `gr(texpreamble="\\usepackage{mathpazo}")` in Julia.
